### PR TITLE
Feature/ZEB-1299/ updated users/invitations tables

### PR DESCRIPTION
**Pull request by MitskevichVS**
Before:
Screen Shot 2020-06-04 at 11 44 58
Screen Shot 2020-06-04 at 11 46 12
Screen Shot 2020-06-04 at 11 46 35
Screen Shot 2020-06-04 at 11 46 25
After:
Screen Shot 2020-06-04 at 11 45 18
Screen Shot 2020-06-04 at 11 45 38
Screen Shot 2020-06-04 at 11 46 53
Screen Shot 2020-06-04 at 11 47 09

### CODE REVIEW

File: client/styles/pages/_users-table.scss [hunk 1]
```diff
@@ -0,0 +1,454 @@
+@import '../../styles/global/var';
+
+.page-wrapper {
+    overflow: hidden;
```
Comment (danymill) [inline client/styles/pages/_users-table.scss:4]: this can affect other pages

File: client/styles/pages/_users-table.scss [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+    .page.user-view-page {
+        padding: 0;
+        padding-top: 0;
+        margin: 80px 20px 20px;
+        position: relative;
+        overflow: hidden;
+        height: 100%;
+
+        .ui-section {
+            height: calc(100% - 100px);
```
Comment (danymill) [inline client/styles/pages/_users-table.scss:15]: 100px?

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/styles/pages/_users-table.scss (hunk 1 above)
Comment (MitskevichVS) [reply client/styles/pages/_users-table.scss]: fixed

File: client/styles/pages/_users-table.scss (hunk 2 above)
Comment (MitskevichVS) [reply client/styles/pages/_users-table.scss]: add comment

**Review by danymill — APPROVED** [review]

**Pull request merged**